index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="stylesheet.css">
+    <title>Sellanni's Webpage</title>
+</head>
+
+<!-- Hero section -->
+<body>
+    <header>
+        <div class="container">
+            <div class="two-column">
+                <div>
+                    <p>Hey, I'm</p>
+                    <H1>Sellanni<br>McKenzie.</H1>
+                    <p class="subtext">My goal is to become a front-end developer, working remotely from the Netherlands.</p>
+                </div>
+                <div>
+                    <img src="sellanni-landing-image.png" alt="Sellanni's profile picture">
+                </div>
+            </div>
+        </div>
+    </header>
+
+<!-- About section -->
+    <section class="light-bg">
+        <div class="container">
+            <h2 class="text-center">About me</h2>
+            <div>
+                <p>
+                    I currently work for the council, within the environmental department. 
+                    My role is interesting and has been great for developing problem solving and interpersonal skills. 
+                    The down side is that it can be super stressful at times (a lot of being screamed at whilst trying to help people).
+                    Before this, I held a job as a software development recruitment consultant, which was great. 
+                    I loved learning about different languages and the projects that people were working on, which has ultimately inspired me to change careers into tech.
+                </p>
+            </div>
+            <div class="two-column">
+                <div style="padding-right: 1rem;">
```

Comment:
Love your use of responsive units instead of hard-coded sizes! It might be a good idea though to have everything in your CSS file rather than some in-line styling so that all your styling is in one place. It looks like everywhere there's inline styling, it's the same `padding-right: 1rem`, so that could become a class.